### DISCUSSION TITLE
All configs for cmp are messed up in plugin_compiled.lua

### DISCUSSION THREAD

**Discussion by hypghzy** [wbthomason/packer.nvim#853] [Q&A]
Neovim: v0.6.1
Os: Manjaro Linux 21.2.4
Germinal emulator: wezterm-git 20220309

I have put [all my config files](https://gitee.com/brucehou/All_configs/tree/SplitVersion/.config/nvim) and compiled file on a git repository, but it seems that the packer_compiled.lua cannot open online. 

I took a [picture ](https://raw.githubusercontent.com/hypghzy/image-hosting/master/01.jpg)which contains the messy code and the error message.

E5112, unexpected symbol shows up every time that I open neovim and all plugins and configs are not working.

Please if anyone can could offer me some advice, since I'm working on that problem for days and getting really anxious about it now.

**Comment by wbthomason** [comment]
Fascinating, I've never seen this before. Can you link me to where in your config you specify `cmp`? Also, what version of Lua/LuaJIT are you using? My best guess is that this is an error with an unterminated string, but I'm not sure how it arises.

**Reply by hypghzy** [reply]
Sorry for the late reply, but the problem was fixed somehow and I have lost the original version of configuration. I guess it was because that  I have some error in the configuration file but it passed the first compile process somehow. There must had some misspelled config option in the config file of cmp. Sorry for the disruption, and thanks for the reply.
